src/Mvc/samples/MvcSandbox/Startup.cs

```diff
@@ -68,6 +68,8 @@ public void Configure(IApplicationBuilder app)
 
                 builder.MapControllers();
                 builder.MapRazorPages();
+
+                builder.MapFallbackToController("Index", "Home");
```

Comment:
example usage.

Reply:
This is neat.
Also you might have forgotten to register the page related stuff on DI

Reply:
yeah, I haven't written tests yet for the MVC cases, and I've only manually tested the controller case.